website/blog/2020-06-21-radium.md

```diff
@@ -0,0 +1,90 @@
+---
+author: Tomasz Godzik
+title: Metals v0.6.2 - Radium
+authorURL: https://twitter.com/tomekgodzik
+authorImageURL: https://avatars1.githubusercontent.com/u/3807253?s=460&v=4
+---
+
+We are excited to announce the release of Metals v0.6.2. The release mostly
+focused on adding bugfixes for the most recent v0.6.1 release, but does include
+several major improvements. This will most likely be the last release before the
+v0.7.0 one.
+
+Metals is a language server for Scala that works with VS Code, Atom, Vim,
+Sublime Text and Emacs. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) along with contributors from the
+community.
+
+Side note, as of this release we also dropped the `Work in progress` label, since Metals already includes most major features of LSP and behaved pretty stable for past few months.
+
+In this release we merged <TODO> PRs and closed <TODO> issues, full details: <TODO>
+
+## TL;DR
+
+- fallback to `show usages` from `Goto definition` if reference is definition
+  itself
+- support for Scala 2.13
+- dropping all Scala versions but 2.11.12, 2.12.7, 2.12.8 and 2.13.0
+- fixes for importing builds in Gradle, Mill and sbt
+- Metals now works with JDK 11
+- miscellaneous fixes
+
+Check out the website and give Metals a try: https://scalameta.org/metals/
+
+## Fallback to `show usages` from `Goto definition`
+
+Thanks to an awesome contribution from @tanishiking we are now able to provide
+you with information about usages if 'Goto definition' was used on the
+definition itself.
+
+![Fallback](https://i.imgur.com/BT3h0FJ.gif)
+
+## Support for Scala 2.13
+
+A lot of work was done by @gabro to make Metals compatible with Scala 2.13. More
```

Comment:
@gabro Anything you would like to add? That was a lot of work and I feel I am simplying it too much.

Also used what you wrote on gitter about dropping older Scala versions - I think it was pretty concise and well explained the situation.

Reply:
I added a more detailed explanation of both what's missing in 2.13 support and also added details about 2.11 deprecation.